jina/peapods/pods/helper.py

```diff
@@ -4,13 +4,13 @@
 
 from ... import __default_host__
 from ...enums import SchedulerType, SocketType, PeaRoleType
-from ...helper import random_port, get_random_identity, get_public_ip, get_internal_ip, is_valid_local_config_source
+from ...helper import random_port, get_random_identity, get_public_ip, get_internal_ip
 
 
 def _set_peas_args(args: Namespace, head_args: Namespace = None, tail_args: Namespace = None) -> List[Namespace]:
     result = []
 
-    for idx in range(args.parallel):
+    for idx in range(getattr(args, 'parallel', 1)):
```

Comment:
this is a smell, parallel should be well defined here

Reply:
thanks. I solved it in a different way now